configurations/ldap-authorization.yaml

```diff
@@ -0,0 +1,5 @@
+authenticator: 'PasswordAuthenticator'
+authenticator_user: cassandra
+authenticator_password: cassandra
```

Comment:
Why are these relevant to this configuration?

Reply:
as we are setting the `authorizer`, and if the user and password weren't set up before, then it need to be set up in here altogether, otherwise, it won't work

Reply:
But this is the authenticator, not autorizer.

Reply:
this is what will allow one user to enter a username and password